hvantk/utils/ucsc.py

```
@@ -0,0 +1,152 @@
+# Utils function to convert USCS single cell dataset to Hail Matrix Tables
+
+import os
+from typing import Union
```

Ruff (0.8.2)

`typing.Union` imported but unused

Remove unused import: `typing.Union`

(F401)